### DISCUSSION TITLE
File extension naming

### DISCUSSION THREAD

**Discussion by aptonline** [erichelgeson/BlueSCSI-v1#91] [General]
I’ve recently purchased a BlueSCSI and loving it so thank you very much for the project.

As I’ve been loading software into images via Basilisk II on my Mac and I’ve had to rename the image extension if the disk images from `.hda` to `.dsk` so that Basillisk II can recognise them, but today I forgot to rename the image back to `.hda` and to my surprise the images mounted and worked as expected.  I did a quick search on here to see if the `.dsk` file extension was supported but couldn’t find any information.

So my question is does BlueSCSI support both and is this a glitch and I should make sure I keep them as `.hda`?

**Comment by erichelgeson** [comment]
In a previous release https://github.com/erichelgeson/BlueSCSI/releases/tag/v1.1-20211215 due to the exact issue you said, the file extension is just ignored as there are no standards! But I see the Usage page doesn't mention that. If you'd like to update it, go ahead, or I'll try to get to it later. Thanks for saying something!
